lib/main/AT32F43x/middlewares/usb_drivers/inc/usb_std.h

```diff
@@ -5,11 +5,11 @@
   **************************************************************************
   *                       Copyright notice & Disclaimer
   *
-  * The software Board Support Package (BSP) that is made available to
-  * download from Artery official website is the copyrighted work of Artery.
-  * Artery authorizes customers to use, copy, and distribute the BSP
-  * software and its related documentation for the purpose of design and
-  * development in conjunction with Artery microcontrollers. Use of the
+  * The software Board Support Package (BSP) that is made available to 
```

Comment:
Lots of unnecessary whitespace changes. Please remove these.

Reply:
max 7456 not working because of lost the default feature OSD in make progress. 

i  fix most the bugs ,add dshot\camera_control\esc_sensors ... in a new branch . a new pr after test fly .